src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -54,25 +56,46 @@ public Site(int id, String name, Boolean enabled, Set<ClientType> types, Set<Str
         this.created = created;
     }
 
+    public Site(int id, String name, String description, Boolean enabled, Set<ClientType> types, Set<String> domains, Boolean visible, long created) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.enabled = enabled;
+        this.clientTypes = types;
+        this.domainNames = domains;
+        this.visible = visible;
+        this.created = created;
+    }
+
     public int getId() { return id; }
     public String getName() { return name; }
+    public String getDescription() { return description; }
```

Comment:
I'm wondering if this should have a default of empty string - and how this would be implemented

Reply:
actually, I am inclined to just leave the description to default to null given this does not seem to be a common pattern we use. besides, this field won't have any functional use.

Reply:
Sounds good

Reply:
flipped on this, I think I'll default to `""`